### PR TITLE
excluded some accounts from the Trending (@hbd.funder))

### PR DESCRIPTION
**Pull request by YaroslavSemeniuk**
Task #2260 - excluded some accounts from the Trending

### CODE REVIEW

File: test/controllers/post/test.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ const {
 const { getNamespace } = require('cls-hooked');
 const { STATUSES } = require('constants/sitesConstants');
 const { FIELDS_NAMES } = require('constants/wobjectsData');
+const _ = require('lodash');
+const { IGNORED_AUTHORS } = require('../../../constants/postsData');
```
Comment (Waiviogit) [inline test/controllers/post/test.js:11]: NODE_PATH .
remove ../../../

File: test/controllers/post/test.js [hunk 2]
```diff
@@ -81,4 +84,51 @@ describe('On postController', async () => {
...
   });
+
+  describe('on getPostsByCategory', async () => {
+    const counter = _.random(2, 9);
+    const postsIds = [];
+    beforeEach(async () => {
+      await dropDatabase();
+      for (let i = 0; i < counter; i++) {
+        const newPost = await PostFactory.Create({
+          author: _.sample([faker.name.firstName().toLowerCase(), IGNORED_AUTHORS[0]]),
```
Comment (Waiviogit) [inline test/controllers/post/test.js:95]: _.sample(IGNORED_AUTHORS)

File: test/controllers/post/test.js [hunk 3]
```diff
@@ -81,4 +84,51 @@ describe('On postController', async () => {
...
+      result = await chai.request(app)
+        .post('/api/posts/')
+        .send({
+          category: 'trending',
+          limit: 10,
+          skip: 0,
+          user_languages: ['en-US'],
+        });
+      result.body.forEach(
+        (element) => expect(element.author).to.be.not.include(IGNORED_AUTHORS[0]),
```
Comment (Waiviogit) [inline test/controllers/post/test.js:116]: (element) => expect(IGNORED_AUTHORS).to.not.include(element.author),

File: test/controllers/post/test.js [hunk 4]
```diff
@@ -81,4 +84,51 @@ describe('On postController', async () => {
...
```
Comment (Waiviogit) [inline test/controllers/post/test.js:130]: (element) => expect(IGNORED_AUTHORS).to.not.include(element.author)

File: test/controllers/post/test.js [hunk 5]
```diff
@@ -7,6 +7,9 @@ const {
 const { getNamespace } = require('cls-hooked');
 const { STATUSES } = require('constants/sitesConstants');
 const { FIELDS_NAMES } = require('constants/wobjectsData');
+const _ = require('lodash');
+const { IGNORED_AUTHORS } = require('../../../constants/postsData');
+const hotTrandGetter = require('../../../utilities/operations/post/feedCache/hotTrandGetter');
```
Comment (Waiviogit) [inline test/controllers/post/test.js:12]: NODE_PATH .
remove ../../../

File: utilities/operations/post/feedCache/hotTrandGetter.js [hunk 6]
```diff
@@ -3,6 +3,7 @@ const config = require('config');
 const { getNamespace } = require('cls-hooked');
 const { redisGetter } = require('utilities/redis');
 const { Post } = require('database').models;
+const { IGNORED_AUTHORS } = require('../../../../constants/postsData');
```
Comment (Waiviogit) [inline utilities/operations/post/feedCache/hotTrandGetter.js:6]: NODE_PATH .
remove ../../../

File: utilities/operations/post/getPostsByCategory.js [hunk 7]
```diff
@@ -10,6 +10,7 @@ const { ObjectId } = require('mongoose').Types;
 const { Post } = require('database').models;
 const { hiddenPostModel, mutedUserModel } = require('models');
 const hotTrandGetter = require('./feedCache/hotTrandGetter');
+const { IGNORED_AUTHORS } = require('../../../constants/postsData');
```
Comment (Waiviogit) [inline utilities/operations/post/getPostsByCategory.js:13]: NODE_PATH .
remove ../../../

File: test/controllers/post/test.js [hunk 8]
```diff
@@ -81,4 +84,51 @@ describe('On postController', async () => {
...
+        });
+        postsIds.push(newPost._id.toString());
+      }
+    });
+    afterEach(() => {
+      sinon.restore();
+    });
+    it('Should return posts without ignored author from Redis', async () => {
+      sinon.stub(hotTrandGetter, 'getTopFromArrays').returns(postsIds);
+      hotTrandGetter.getTopFromArrays(postsIds);
```
Comment (Waiviogit) [inline test/controllers/post/test.js:105]: what is it??? remove

File: test/controllers/post/test.js [hunk 9]
```diff
@@ -81,4 +84,51 @@ describe('On postController', async () => {
...
+
+      result = await chai.request(app)
+        .post('/api/posts/')
+        .send({
+          category: 'trending',
+          limit: 10,
+          skip: 0,
+          user_languages: ['en-US'],
```
Comment (Waiviogit) [inline test/controllers/post/test.js:113]: remove

File: test/controllers/post/test.js [hunk 10]
```diff
@@ -81,4 +84,51 @@ describe('On postController', async () => {
...
+    it('Should return posts without ignored author from MongoDB', async () => {
+      result = await chai.request(app)
+        .post('/api/posts/')
+        .send({
+          category: 'trending',
+          limit: 10,
+          skip: 0,
+          user_languages: ['en-US'],
+        });
+      result.body.forEach(
```
Comment (Waiviogit) [inline test/controllers/post/test.js:129]: in this case you have result body [] 
add check for body length for both cases

**Review by Fesmofet — APPROVED** [review]

**Pull request merged**